app/com/arpnetworking/metrics/portal/reports/impl/chrome/BaseScreenshotRenderer.java

```diff
@@ -107,7 +109,7 @@
                 .addData("format", format)
                 .addData("timeRange", timeRange)
                 .log();
-        final CompletableFuture<Void> navigate = dts.navigate(getUri(source).toString());
+        final CompletableFuture<Void> navigate = dts.navigate(getUri(source, timeRange).toString());
```

Comment:
Just want to confirm that the `timeRange` here is the fixed target and independent of any scheduling and jitter. (e.g. a daily midnight to midnight report with a 1h delay will still have a timerange of midnight to midnight)